src/window_base.cpp

```diff
@@ -40,6 +40,22 @@ Window_Base::Window_Base(int x, int y, int width, int height) {
 	SetHeight(height);
 	SetStretch(Game_System::GetMessageStretch() == RPG::System::Stretch_stretch);
 	SetZ(Priority_Window);
+
+	current_frame = -1;
+	total_frames = -2;
```

Comment:
> 2.3: RM2000 + RM2003: The battle command window (both left [manual, 

I assume the value is "-2" to ensure that ``IsMovementActive`` returns false. Imo they could be both set to "-1" and ``MovementActive`` gets an extra check for ">= 0"

Reply:
Initialized both to 0 and added a check to `IsMovementActive()`